src/software/ai/hl/stp/tactic/goalie_tactic.cpp

```diff
@@ -264,8 +264,8 @@ void GoalieTactic::calculateNextAction(ActionCoroutine::push_type &yield)
 
             // we want to restrict the block cone to the friendly crease, also potentially
             // scaled by a defense_area_deflation_parameter
-            Rectangle deflated_defense_area = field.friendlyDefenseArea();
-            deflated_defense_area.expand(-defense_area_deflation);
+            Rectangle deflated_defense_area =
+                field.friendlyDefenseArea().scale(-defense_area_deflation);
```

Comment:
I could be mistaken, but I'm pretty sure `scale` multiplies the area instead of adding to it, so this is very much not the same?

Reply:
I believe it's additive